scripts/formvalidation.js

```diff
@@ -0,0 +1,11 @@
+const contactForm = document.querySelector('.contactForm');
+const emailImput = document.querySelector('.emailAddress');
+const validationMessage = document.querySelector('.validationMessage');
+
+contactForm.addEventListener('submit', (event) => {
+  const stringMessage = emailImput.value;
+  if (/[A-Z]/.test(stringMessage)) {
+    validationMessage.innerHTML = 'Your email is supposed to be lower cased.';
+    event.preventDefault();
+  }
```

Comment:
- [ ] **Required:** You have done an amazing job so far, however, your code submits with uppercase letters present. You could use an event listener other than relying on the default `HTML` form submit.